LiveStream/Service/KsLive.swift

```diff
@@ -0,0 +1,39 @@
+import FirebaseAnalytics
+import FirebaseDatabase
+import FirebaseAuth
+
+public class KsLiveApp {
```

Comment:
actually ended up deleting and moving stuff to the `LiveStreamServiceProtocol` which cleans up things!